### PR TITLE
header+burger v0.1 beta

### PR DESCRIPTION
**Pull request by SoloOleh**
Недороблені стилі та функціонал

### CODE REVIEW

File: src/css/header.css [hunk 1]
```diff
@@ -1,4 +1,291 @@
+@import url('./font.css');
```
Comment (hrebynakha) [inline src/css/header.css]: dont need this, it's alredy imported
image

File: src/css/header.css [hunk 2]
```diff
@@ -1,4 +1,291 @@
...
+
 .header {
-  padding: 20px 0;
-  font-weight: 600;
+    padding: 20px 0;
+    font-family: "DM Sans", sans-serif;
```
Comment (snegurova) [inline src/css/header.css]: font family had been already set

File: src/js/burger-menu.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const burgerMenuButton = document.querySelector('.burger-menu');
+  const burgerMenuContent = document.querySelector('.burger-menu-content');
+  const closeMenuButton = document.querySelector('.close-menu');
+  const navLinks = document.querySelectorAll('.router-link');
+
+  // За замовчуванням щоб був Home активний
```
Comment (snegurova) [inline src/js/burger-menu.js]: If you keep comments please keep them in English

File: src/main.js [hunk 4]
```diff
@@ -6,3 +6,5 @@ links.forEach(link => link.addEventListener('click', route));
 
 handleLocation();
 updateQuote();
+
+import './js/burger-menu';
```
Comment (snegurova) [inline src/main.js]: Please move import on top of the file

File: src/partials/header.html [hunk 5]
```diff
@@ -1,14 +1,107 @@
...
+    <button class="burger-menu">
+      <svg class="icon icon-burger" width="32" height="32">
+        <use href="../img/icons-header.svg#icon-burger"></use>
+      </svg>
+    </button>
     <nav class="nav">
       <ul class="navbar-list">
         <li class="navbar-item">
-          <a href="/" class="router-link">Home</a>
+          <a href="../pages/home.html" class="router-link">Home</a>
```
Comment (snegurova) [inline src/partials/header.html]: link should be to the page '/'

File: src/partials/header.html [hunk 6]
```diff
@@ -1,14 +1,107 @@
...
         </li>
         <li class="navbar-item">
-          <a href="/favorites" class="router-link">Favorites</a>
+          <a href="../pages/favorites.html" class="router-link">Favorites</a>
```
Comment (snegurova) [inline src/partials/header.html]: link should be to the page '/favorites'

File: src/partials/header.html [hunk 7]
```diff
@@ -1,14 +1,107 @@
...
+  <div class="burger-menu-content">
+    <button class="close-menu">
+      <svg class="icon icon-close" width="32" height="32">
+        <use href="../img/icons-header.svg#icon-close"></use>
+      </svg>
+    </button>
+    <nav class="burger-nav">
+      <ul>
+        <li><a href="../pages/home.html">Home</a></li>
+        <li><a href="../pages/favorites.html">Favorites</a></li>
```
Comment (snegurova) [inline src/partials/header.html]: link should be to the page `'/'` and `'/favorites'`
Please add class `.router-link`. This class is used only for router, not for styling

File: src/css/header.css [hunk 8]
```diff
@@ -1,4 +1,291 @@
...
+    .navbar-list {
+        width: 233px;
+        height: 52px;
+        flex-shrink: 0;
+        justify-content: space-between;
+        padding: 4px;
+        margin: 0;
+    }
+
+    .router-link {
```
Comment (snegurova) [inline src/css/header.css]: This class is used only for router, not for styling

**Review by snegurova — APPROVED** [review]

**Pull request merged**